test/e2e/docs/style-guide-playwright.md

```diff
@@ -124,11 +136,13 @@ export class SomeComponent extends BaseContainer {
 
 ## Page Objects
 
-Page objects are to be used to represent a corresponding page on WPCOM. It can hold element selectors, class methods to interact with the page and define other helper functions.
+Page objects are to be used to represent a corresponding page on WPCOM. It can hold attributes, class methods to interact with the page and define other helper functions.
 
 A well-implemented page object will abstract complex interactions on the page to an easily understandable method call. The method should be well-contained, predictable and easy to understand.
 
-Every page object file should contain an object outside of the class definition to hold element selectors. The Page object should access element selector values using dot notation within the method calls.
+- **Don't Repeat Yourself (DRY)**: common actions can be called from the page object.
+- **maintainability**: if a page changes, update the page object at one spot.
+- **readability**: named variables and functions are much easier to decipher than series of strings.
 
 On many pages of WPCOM elements will load asynchronously. This leads to issues when initializing page objects as constructors cannot be asynchronous. To address this, page objects almost always inherit from the `BaseContainer` class as it provides asynchronous initialization of the page object through use of static method `Expect`. Only use synchronous class constructor if the page in question does not require any post-initialization setup.
```

Comment:
Do we want to mention the break away from this in Playwright?

Reply:
Yep, removed the paragraph. Thanks!